tutorials/en/vectara-advanced-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-or-customer-support-use-case.mdx

```diff
@@ -0,0 +1,671 @@
+
+The beauty of Vectara lies in its API-addressable platform, which is a canvas for developers to paint their search solutions, embedding them within their applications.
+
+---
+### 🎨 Dive into Vectara's Console
+
+To truly grasp the potential of Vectara, let's delve into its console which is the epicenter of managing your search ecosystem:
+
+- **Creating Corpora**: Start by creating a corpus, a sanctuary for your data awaiting to be queried. The process is straightforward - name your corpus, describe it, select an embedding model, specify filter attributes, and voila, your corpus is ready to be fed with data.
+- **API Access Management**: Vectara grants you the wand to manage API access. Create and manage API keys and app clients with ease. With the necessary permissions, an `API access` tab unveils in the sidebar, guiding you to create API keys and app clients. It's your doorway to interact with Vectara's treasure trove of search capabilities.
+- **Team Collaboration**: Invite your team to the Vectara console, assign specific roles, and foster a collaborative environment to build and refine your search solutions.
+- **Billing Management**: Keep a tab on your account usage and manage billing details ensuring uninterrupted service as you sail through the Vectara ecosystem.
+
+
+In this section, we've skimmed the surface of Vectara's offerings. As we delve deeper into our chosen use case in the next section, the utility and power of Vectara will unfold further, painting a clearer picture of how it can be harnessed for Customer Support applications.
+
+---
+## Our Quest: Orchestrating a Customer Support Maestro 🎵
+
+Embarking on the expedition to create a Customer Support solution requires a sturdy vessel, and Vectara proves to be an admirable choice for this voyage. Unlike the conventional techniques that revolve around wrapping OpenAI APIs without delving deeper into the core concepts, Vectara emerges as a beacon of innovation especially in the wake of GPT-4's multi-modal capabilities which sent ripples through the RAG-based companies recently. Vectara simplifies the creation of QA and conversational based projects, extending its arms to developers with its pre-built models and custom workflows. This eradicates the superficial approach often taken by budding developers who get entangled in ready-made wrappers like Langchain, without a thorough understanding of the underlying principles and optimizations.
+
+### 🎯 Why Vectara for Customer Support?
+
+Vectara shines bright by doing the heavy lifting, encapsulating best practices, and providing both REST and gRPC APIs. This level of customization is a realm beyond, offering unparalleled control while ensuring speed, accuracy, and quality in responses. It's like having a seasoned sailor aboard who knows the ropes, ensuring a smooth sail amidst the turbulent seas of development.
```

Comment:
The analogy with sailing is maybe too much here. Just my pov as a reader. Overused :)
I am not sure how the REST/gRPC comment is connected. I think focusing on "doing the heavy lifting" and "encapsulating best practices" is more relevant and maybe add more details?

Reply:
How does this sound to you : 

🎯 Why Vectara for Customer Support?
Vectara stands out in the realm of customer support by taking on the complex challenges and intricacies of development, effectively doing the heavy lifting for your team. By encapsulating industry best practices within our solutions, we ensure that you’re always on the cutting edge, providing fast, accurate, and high-quality responses to your customers.

Vectara is versatile, offering a range of integration options through both REST and gRPC APIs. This ensures that regardless of your technical setup or preferences, implementing and scaling Vectara within your customer support workflow is seamless and efficient.